src/server/api/controllers/create-user-roles.controller.js

```diff
@@ -0,0 +1,14 @@
+const knex = require('../../config/db');
+
+// Creating user roles in user_roles table
+const createUserRoles = async (body) => {
+  await knex('user_roles').insert({
+    fk_role_id: body.fk_role_id,
```

Comment:
some warnings. Can you handle those?

Reply:
The warning are related to camel case naming which I heard one of our classmates is fixing it with a PR.